TempMail/utils/Email.swift

```diff
@@ -32,12 +32,16 @@ class Email : ObservableObject{
         let letters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
         //      returns a random permutation of letters; string of up to length 8 for email addr
         self.email_addr = String((0..<8).map{ _ in letters.randomElement()!})
+//        reset inbox
+        self.inbox = [InboxModel]()
```

Comment:
This is bad because of side effects. Make a reset() function that's called when the button is pressed that sets a new email addr and sets the inbox to nil. Since we can use [InboxModel]? it makes more sense to do that instead of assigning it to be an empty array. We can also change the return type of setEmailAddr from a void to a String and set self.email_addr in the rest function